src/app/onion/learning-object-builder/components/editor-action-panel/change-status-modal/change-status-modal.component.scss

```diff
@@ -0,0 +1,34 @@
+@import '~_vars.scss';
+
+.status-options-wrapper {
+  margin-top: 25px;
+  min-width: 30vw;
+}
+.status-item {
+  display: flex;
+  flex-flow: row nowrap;
+  margin-bottom: 15px;
+  box-shadow: 0 0 5px lighten($placeholder, 10);
+  padding: 8px 24px;
+  cursor: pointer;
+  &.unfocused {
+    opacity: .3;
+  }
+  &:hover {
+    box-shadow: 0 0 5px lighten($bright-blue, 10);
+  }
+  .status-item__text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 15px;
+  }
+}
+
+.btn-group {
+  .good {
+    background: $bright-blue;
+    border-color: $bright-blue;
+  }
+  margin-top: 25px;
```

Comment:
In general, it's a good idea to put top level rules over nested selectors. IE, this should read:
```
.btn-group {
    margin-top: 25px;

    .good { ... }
}
```